docs/installation.rst

```diff
@@ -71,7 +71,6 @@ instead of ``pip``:
     conda install bottleneck
     conda install -c conda-forge notebook
     conda install -c conda-forge jupyterlab
-    conda install -c conda-forge voila
```

Comment:
Don't you need to add a line for solara install here?

Reply:
we can either add it (making sure its available on conda-forge first) or wait for any sign that there are problems installing from pip that are resolved by installing through conda? 🤷‍♂️

Reply:
I don't think we need to add it here yet (if it's even installable via conda).